specification/resources/resource-manager/Microsoft.Solutions/stable/2019-07-01/managedapplications.json

```diff
@@ -1606,16 +1606,39 @@
           "type": "string",
           "description": "The identity type.",
           "enum": [
-            "SystemAssigned"
+            "SystemAssigned",
```

Comment:
will this cause breaking changes

Reply:
This shouldn't be breaking. It's adding new value to enum and added a not required property. Old schema should still be accepted.

Reply:
Do you have SDK released for this? As the enum is not modeled as string, new SDK released will be broken because the values in enum could be different